docker_scripts/build.sh

```diff
@@ -0,0 +1,7 @@
+name=axon
+
+name_with_tag="wpilib/${name}:latest"
+
+docker build  -t ${name} ..
+docker tag ${name} ${name_with_tag}
+docker push ${name_with_tag}
```

Comment:
We should not assume users have push access.

Reply:
This file is for developer use only. Perhaps I should move it somewhere else, but probably just remove it since GitHub actions does it anyway.